Resources/Prototypes/Entities/Objects/Specific/Medical/ivbag.yml

```diff
@@ -0,0 +1,195 @@
+##
+##    IV Bags & Stands
+##
+
+
+#     IV Bag
+
+- type: entity
+  parent: BaseItem
+  name: iv bag
+  id: BaseIVBag
+  # description: An injectable IV bag that slowly drips its contents.
+  description: An injectable IV bag. Holds plenty of fluid.
+  components:
+  - type: Item
+    sprite: Objects/Specific/Medical/ivbag.rsi
+    HeldPrefix: bag
+    size: 10
+  - type: Sprite
+    sprite: Objects/Specific/Medical/ivbag.rsi
+    layers:
+      - state: bag_chem
+      - state: fluid1
+        map: ["enum.SolutionContainerLayers.Fill"]
+        visible: false
+  - type: Appearance
+    visuals:
+    - type: SolutionContainerVisualizer
+      maxFillLevels: 10
+      fillBaseName: fluid
+  - type: SolutionContainerManager
+    solutions:
+      injector:
+        maxVol: 200
+  - type: Injector
+    injectOnly: false
```

Comment:
right now this won't inject into people's bloodstreams; it'll inject into their chemstream, since thats the `InjectableSolution` for mobs. this behavior specifically probably just shouldn't be in yet until it can be properly done (through a custom component)

Reply:
Oh, good catch. Should be easy to add an override to the Injector component to only target bloodstreams, but if the only use is for IV bags then a the new component may be the way to go before merging.